### PR TITLE
use travis for ci testing

### PR DESCRIPTION
**Pull request by cz8s**
snap is not helpful anymore. I again had to spend hours to fix snap to be able to make the last pr green.
This is not complete yet. The functional tests and the deploy are missing.
But comments of how you think we should proceed would be apreciated :)

### CODE REVIEW

**Comment by lplotni** [comment]
What is the exact problem with Snap? I haven't touch it in a while and besides the heroku-deployment being broken, I only have the feeling that for whatever reason it got slower. I don't mind switching to Travis as long as we have similar setup - possible?

File: .travis.yml [hunk 1]
```diff
@@ -11,5 +11,9 @@ before_install:
   - docker ps -a
 before_script:
   - npm install -g gulp
+  - gulp selenium-install
   - ./node_modules/db-migrate/bin/db-migrate up
-script: gulp test && gulp test-integration
+script:
```
Comment (lplotni) [inline .travis.yml]: This will execute all tests, not only functional right?

Comment (cz8s) [inline .travis.yml]: the second commit makes it more : https://github.com/lplotni/pace/pull/253/commits/d5db0058b29afc950bb6d2c694b548b3d4060f9d

**Comment by cz8s** [comment]
Every fork of our code should run the same tests as our master. But this is not the case.
For example all forks still run 'gulp create-db' instead of the current 'gulp create-pace-db'
And I don't know why. This means that all forks are red and it is not easy to merge them.

**Comment by cz8s** [comment]
the nice thing about travis is, that everyone can see exactly what our pipeline does.

**Pull request merged**